README.md

```diff
@@ -1,156 +1,132 @@
-# Forever and Ever (FAE)
-"Forever & Ever" is a mod for the free game [Doki Doki Literature Club](https://www.ddlc.moe) from [Team Salvato](http://teamsalvato.com/). In it, you can become closer to Sayori than ever before! You have all the time in the world to build the perfect relationship with our cinnamon bun as the star of the show. At the heart of the mod, we took a mental health based approach in line with the experiences Sayori had in the original game, so we can hopefully support people by offering a source of positivity, advice, and unconditional understanding. 
-Currently we have a whole range of unique dialogue, questions, and games for you and Sayori to enjoy, and we have so many more ideas for the near future! We're very active around the mod and on our discord server and we promise to deliver frequent updates and announcements so there's always something new and exciting to do! 
-Why not get creative and try to add a feature or two yourself? (I'm sure she'd love that!)
-
-Please check the [Releases](https://github.com/ForeverAndEverTeam/fae-mod/releases) page for the latest stable build.
-Also, please don't clone the master branch to download the mod. 
-Use a stable release instead!
-
-## СOMPATIBILITY NOTICE 
-This mod is incompatible with Doki Doki Literature Club Plus. 
-If you own a DDLC+ copy, please download the original game from it's [official website](https://ddlc.moe) to install the mod.
-
-## REPORTING BUGS
-Please report bugs on our [issues](https://github.com/ForeverAndEverTeam/fae-mod/issues) page.
-Please be as detailed as possible when reporting, to make our lives easier when fixing.
-
-<details>
-  <summary>NEW CREDITS</summary>
-  
-## DIRECTOR
-
-[Orcaramelo](https://github.com/Orcaramelo)
-
-## PROGRAMMING
-
-[Nathan](https://github.com/TRIDENT1313) (Lead)
-
-## ART
-
-[Orcaramelo](https://github.com/Orcaramelo) (Lead), [Nemu-sus](https://github.com/Nemu-sus)
-
-## MUSIC
-
-SlightlyAmiss (Theme), [Multimokia](https://github.com/multimokia) (Music box loop)
-
-## WRITING
-
-Sayori, [Orcaramelo](https://github.com/Orcaramelo), [Emilia/Monika (EYLO)](https://www.reddit.com/u/EmiliaMonika?utm_medium=android_app&utm_source=share), [Otter](https://github.com/my-otter-self), [Nathan](https://github.com/TRIDENT1313), Sobb
-
-</details>
+<p align="center">
+    <img src="https://justnatsuki.club/img/guest/fae/forever_and_ever_logo.png" height="300"/>
```

Comment:
That's fine, but the purpose of the logo is for appearances and polish. Having and displaying (some) branding helps reinforce this is the official GitHub page of the project, and helps display a level of polish and pride in the mod itself. I'd note that this isn't unique to JN: MAS also displays their logo prominently, and I'd argue if the branding already exists, it makes sense to use it!

Up to you how you'd like to handle it; it's your project after all - it can be swapped out for something of your choice, or removed entirely: I'd personally recommend retention given the above.

Reply:
Yeah fair point.
Might as well add it in then.